onyo/lib/commands.py

```diff
@@ -1039,7 +1039,7 @@ def onyo_set(inventory: Inventory,
 @raise_on_inventory_state
 def onyo_tree(inventory: Inventory,
               dirs: list[tuple[str, Path]]) -> None:
-    """Print the directory tree of paths.
+    r"""Print the directory tree of paths.
```

Comment:
Here is a good example for a different way of phrasing the tree thing along onyo texts.
Is it the directory tree, file tree? tree-like output (like stated by docstring of `def _tree()` below), or "in a tree like format"?